ja2003.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.github.christianrauch.Jahresarbeit-2003</id>
+  
+  <name>Jahresarbeit 2003</name>
+  <summary>Find die größte Fläche von zusammenhängenden Quadraten</summary>
+
+  <url type="homepage">https://github.com/christianrauch/Jahresarbeit-2003</url>
+  
+  <project_license>GPL-2.0</project_license>
+  <metadata_license>OFL-1.1</metadata_license>
+  <metadata_license>Apache-2.0</metadata_license>
+  
+  <recommends>
+    <display_length compare="ge">1024</display_length>
+  </recommends>
+  
+  <description>
+    <p>
+      Das Programm &quot;Jahresarbeit 2003&quot; entstand als Jahresarbeit im Fach Informatik. Nach Spielstart sieht man ein 9x9 Feld mit verschiedenfarbigen Quadraten. Am Rand des Feldes sind Pfeile zu erkennen, welche genutzt werden um die Reihen und Spalten in denen sich die Quadrate befinden zu verschieben. Zusammenhängende gleichfarbige Quadrate bilden eine Fläche, welche durch draufklicken entfernt wird. Ziel des Spieles ist es eine möglichst große Fläche in kürzester Zeit weg zu klicken. Das Spiel ist zu Ende wenn alle Quadrate entfernt sind oder es keine Möglichkeit mehr gibt um die restlichen Quadrate zu entfernen. Jedoch sollte man um alle Quadrate zu entfernen so wenig Klicks wie möglich gebrauchen, da diese die Endpunktzahl verringern.
+    </p>
+  </description>
+  
+  <launchable type="desktop-id">com.github.christianrauch.Jahresarbeit-2003.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Hauptmenü</caption>
+      <image>screenshots/menu.png</image>
```

Comment:
screenshot should be accessible from a public URL. This will fail

Reply:
Well, they are part of this PR. So they will be public once it is merged and a repo was created. I don't know how I can reference them otherwise.

Reply:
they have to be available at the time of the build.

Reply:
When the flatpak repo is checked out during build time, the files will be available locally, just like the icon or desktop file. How can I reference those files in the XML?